src/book/03-guides/04-general/01-learning-resources.mdx

```diff
@@ -0,0 +1,392 @@
+- [Official Gatsby Documentation](https://www.gatsbyjs.com/docs/)
+- [Official Tailwind Documentation](https://tailwindcss.com/docs)
+
+</details>
+
+<details>
+<summary> Web Development (Internal) </summary>
+
+**Reading:**
+
+- [NUbook: NUsight](/system/tools/nusight)
+
+**Resources:**
+
+- [GitHub: NUsight](https://github.com/NUbots/NUbots/tree/master/nusight2)
+- [GitHub: NUbook](https://github.com/NUbots/NUbook)
+
+</details>
+
+<details>
+<summary> Documentation </summary>
+
+Since NUbots is a student organisation, high turnover is unavoidable. To help combat the issues this creates, good documentation is key to the success of the team. This includes low level documentation in the form of code comments and high level documentation in the form of NUbook pages.
+
+**Reading:**
+
+- [NUbook: NUbook](/system/tools/nubook)
+- [Australian Network on Disability: A Beginner's Guide to Accessible Content](https://www.and.org.au/pages/a-beginners-guide-to-accessible-content.html)
+
+**Tutorials:**
+
+- [Google Technical Writing Course](https://developers.google.com/tech-writing) - free, in-class sessions are open to the public!
+
+</details>
+
+## Hardware
+
+<details>
+<summary>Soccer Robot</summary>
+
+**Reading:**
+
+- [NUbook: Hardware Overview and Specifications](/system/hardware/overview)
+- [NUbook: Input](/system/subsystems/input)
+- [NUbook: Flashing a Robot](/guides/hardware/flashing)
+
+**Resources:**
+
+- [GitHub: NUware](https://github.com/NUbots/NUware)
+
+</details>
+
+## Localisation and Odometry
+
+<details>
+<summary> Particle Filters </summary>
+
+**Videos:**
+
+- [YouTube: Introduction to Autonomous Navigation](https://www.youtube.com/watch?v=Fw8JQ5Q-ZwU)
+- [YouTube: Understanding the Particle Filter](https://www.youtube.com/watch?v=NrzmH_yerBU)
+- [YouTube: Monte Carlo Localisation and Particle Filters](https://www.youtube.com/watch?v=MsYlueVDLI0&t=1608s)
+
+</details>
+
+<details>
+<summary> Simultaneous Localisation and Mapping (SLAM) </summary>
+
+**Videos:**
+
+- [YouTube: Introduction to SLAM](https://www.youtube.com/watch?v=saVZtgPyyJQ)
+
+</details>
+
+## Motion
+
+<details> 
+<summary>General</summary>
+
+**Reading:**
+
+- [NUbook: Motion](/system/subsystems/motion)
+- [NUbook: Running and Tuning Scripts](/guides/main/tuning-and-running-scripts)
+
+**Resources:**
+
+- [GitHub: Simscape](https://github.com/NUbots/Simscape)
+
+</details>
+
+## Vision
+
+<details>
+<summary>General</summary>
+
+**Reading:**
+
+- [NUbook: Vision](/system/subsystems/vision)
+- [NUbook: Maintaining Subsystems # Vision](/guides/main/maintaining-subsystems#vision)
+
+</details>
+
+<details> 
+<summary>Visual Mesh</summary>
+
+**Reading:**
+
+- [NUbook: Visual Mesh Getting Started](/guides/tools/visualmesh)
+
+**Resources:**
+
+- [GitHub: Visual Mesh](https://github.com/Fastcode/VisualMesh)
+
+</details>
+
+<details>
+<summary>Data Collection</summary>
+
+**Reading:**
+
+- [NUbook: NUpbr](/system/tools/nupbr)
```

Comment:
I suppose you don't think NUgan is mature enough to link here?

Reply:
I was considering it, but we don't really use it (yet).